Codeinwp/neve
# fix: bring back custom font size field | remove the preset sizes style

fix: bring back custom font size field [closes #3173]
chore: drop custom sizes for editor presets [roll back #3148]

### Summary
- Adds back the custom font-size input inside the editor;
- Removes styles we applied to the preset sizes inside the editor as they were messing up some user sites; They also broke the TPC templates that could be inserted inside the editor;


### Will affect visual aspect of the product

Depends if they were previously used.

### Test instructions


- Go into the editor, add a paragraph block;
- There should be a custom size input for font size;
- If you use any of the presets (Huge, Large, Small, etc), these should have font sizes applied by the core, not from the theme;


Closes #3173.



cc: @Codeinwp/design-team just keeping you in the loop 🚀
